src/routers/oauth2Callbacks.ts

```diff
@@ -0,0 +1,90 @@
+import express, { Request, Response } from 'express';
+import { handleExpressError } from './standardError';
+import { logger } from '../utils/logger';
+import { SOCIAL_NETWORKS } from '../entities/socialProfile';
+import { oauth2CallbackHandler } from '../services/socialProfileService';
+import { findProjectVerificationFormById } from '../repositories/projectVerificationRepository';
+
+export const oauth2CallbacksRouter = express.Router();
+
+const successPagePath = `${process.env.GIVETH_IO_DAPP_BASE_URL}`;
+export const SOCIAL_PROFILES_PREFIX = '/socialProfiles';
+const generateDappVerificationUrl = async (params: {
+  projectVerificationId: number;
+  url: string;
+}): Promise<string> => {
+  const { projectVerificationId, url } = params;
+  const projectVerificationForm = await findProjectVerificationFormById(
+    projectVerificationId,
+  );
+  return `${successPagePath}/${projectVerificationForm?.project?.slug}`;
```

Comment:
I feel this might URL might be a bit too general. But since we are not doing tons of callbacks from different services in different parts of the dapp. This is ok for now.

For the future we might have to change it. To something more specific. But nothing too too important now.

Reply:
To deliver. Lets leave it like that.

Reply:
Yeah I agree with you , and this is what Alireza sent to me to redirect to that, if frontend DEVs specify another path we can redirect to that